coalib/coala_main.py

```diff
@@ -20,6 +20,21 @@ def do_nothing(*args):
     return True
 
 
+ALL_ACTIONS_POSSIBLE = ['Do (N)othing', '(A)pply patch', 'Add (I)gnore comment',
```

Comment:
Well, you can create a method that does it and stores it into a variable and when it retriggers again we can just give them the variable.